api/src/services/authentication.ts

```diff
@@ -294,7 +294,6 @@ export class AuthenticationService {
 				user_role: 'u.role',
 				share_id: 'd.id',
 				share_item: 'd.item',
```

Comment:
`share_item`,`share_collection`,share_times_used` and `share_max_users` do not seem to be used anymore either?